### ISSUE TITLE
Add support for stack.yaml in addition to stack.yml

### ISSUE THREAD

**Issue by ad-si** [openfaas/faas-cli#666]
According to the docs the official file extension is `.yaml` (https://yaml.org/faq.html) and therefore it should be supported!

**Comment by alexellis** [comment]
Let's monitor this suggestion to see how much demand there is.

**Comment by alexellis** [comment]
Thanks for your interest. Please could you have a look at the various different repos we have and what each does?

**Comment by ad-si** [comment]
> Thanks for your interest. Please could you have a look at the various different repos we have and what each does?

Sorry, I'm used to monorepos. Will be more considerate in the future 😅

**Comment by alexellis** [comment]
You're right about the website, but practice says different: https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml

I think it will remain as stack.yml unless we have a significant reason to alter it. Engineering effort and maintenance doesn't come for free. 

Thanks for the suggestion though. Perhaps some of these would be better as conversational comments on Slack?

**Comment by martindekov** [comment]
I believe we already have this covered. It can be see in [faas.go](https://github.com/openfaas/faas-cli/blob/7af2864d6d94f2a676d478c4698e4ef1795508f8/commands/faas.go#L67).

**Comment by martindekov** [comment]
It should be slightly augmented in order to work, I guess if needed can be easily done.

**Comment by alexellis** [comment]
I think that reference is for the auto-completion.

**Comment by MrWormHole** [comment]
I am summoning this issue back again :P we should have yaml first supported then yml if both files are given at the same time, let's use yaml? I checked it but it looks very straight forward to add. YAML/YML are both the same btw so it makes sense to support yaml.
![yml-yaml](https://user-images.githubusercontent.com/22800416/130755264-f28f7418-8e80-42f8-a264-34b7b08f5222.png)
